### PR TITLE
Mon 34072 move opt to common

### PR DESCRIPTION
**Pull request by bouda1**
REFS: MON-34072

### CODE REVIEW

File: engine/src/retention/contact.cc [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
 #include "com/centreon/engine/retention/contact.hh"
+#include "com/centreon/common/opt.hh"
```
Comment (jean-christophe81) [inline engine/src/retention/contact.cc:21]: yet in contact.hh

File: engine/src/retention/contact.cc [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
 #include "com/centreon/engine/string.hh"
 
+using com::centreon::common::opt;
```
Comment (jean-christophe81) [inline engine/src/retention/contact.cc:24]: yet in contact.hh

File: engine/src/retention/host.cc [hunk 3]
```diff
@@ -25,8 +24,8 @@
 #include "com/centreon/engine/logging/logger.hh"
 #include "com/centreon/engine/string.hh"
 
+using com::centreon::common::opt;
```
Comment (jean-christophe81) [inline engine/src/retention/host.cc:27]: yet in host.hh

File: engine/src/retention/program.cc [hunk 4]
```diff
@@ -1,26 +1,26 @@
...
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
 #include "com/centreon/engine/retention/program.hh"
 
 using namespace com::centreon::engine;
 using namespace com::centreon::engine::retention;
+using com::centreon::common::opt;
```
Comment (jean-christophe81) [inline engine/src/retention/program.cc:23]: idem

File: engine/src/retention/service.cc [hunk 5]
```diff
@@ -25,8 +24,8 @@
 #include "com/centreon/engine/logging/logger.hh"
 #include "com/centreon/engine/string.hh"
 
+using com::centreon::common::opt;
```
Comment (jean-christophe81) [inline engine/src/retention/service.cc:27]: idem

**Comment by bouda1** [comment]
@coderabbitai review

**Review by jean-christophe81 — APPROVED** [review]

**Pull request merged**